.travis.yml

```diff
@@ -1,19 +1,23 @@
 language: node_js
-node_js: '6'
+node_js: stable
 addons:
   firefox: latest
+  sauce_connect: true
```

Comment:
Hmm, I haven't seen this in any of the other PolymerElement repos. Just thought I would call it out as an inconsistency.